cloudflare/src/endpoints/account/mod.rs

```diff
@@ -15,9 +15,9 @@ pub struct Account {
     /// Account name
     pub name: String,
     /// Account Settings
-    pub settings: Settings,
+    pub settings: Option<Settings>,
```

Comment:
Rather than make Settings optional, perhaps it should implement the Default trait?

Reply:
Implementing Default on Settings may not correctly reflect the values returned by an endpoint that does include the settings field, but someone using this library that doesn't know that might try to depend on the default values of the Settings field despite them being incorrect for their particular use case.